schemachange/module.mk

```diff
@@ -17,7 +17,8 @@ schemachange/sc_records.o	\
 schemachange/sc_schema.o	\
 schemachange/sc_stripes.o	\
 schemachange/sc_struct.o	\
-schemachange/sc_util.o
+schemachange/sc_util.o      \
+db/comdb2uuid.o
```

Comment:
Yes, by why do you need the obj file in this library? Just the include should be sufficient and finally the linker will find and resolve references when building `comdb2` binary.